packages/account/src/Types/common-prop.type.ts

```diff
@@ -70,3 +71,22 @@ export type TApiContext = {
 };
 
 export type TPopoverAlignment = 'top' | 'right' | 'bottom' | 'left';
+
+export type TRoute = {
+    default?: boolean;
+    exact?: boolean;
+    id?: string;
+    icon_component?: string;
+    is_invisible?: boolean;
+    path?: string;
+    to?: string;
+    component: ((cashier_routes?: TRoute[]) => JSX.Element) | typeof Redirect;
+    getTitle: () => string;
+    subroutes?: TRoute[];
```

Comment:
@likhith-deriv it seems that icon is missing here, also the nesting of TRouteConfig should correspond to this array of objects:
<img width="859" alt="Screenshot 2022-11-04 at 12 50 29" src="https://user-images.githubusercontent.com/103177211/199944018-44bc0f41-b948-4687-895f-a8d99818ae71.png">

Reply:
Will make the change